src/UploadedFile.php

```diff
@@ -80,7 +80,7 @@ public function __construct($streamOrFile, $size, $errorStatus, $clientFilename
 
         if (\UPLOAD_ERR_OK === $this->error) {
             // Depending on the value set file or stream variable.
-            if (\is_string($streamOrFile)) {
+            if (\is_string($streamOrFile) && '' !== $this->file) {
```

Comment:
I think `$this->file` will always be `null` here, right?

Reply:
right, fixed!